src/canvas-driver.js

```diff
@@ -79,9 +79,17 @@ function renderElement (context, element, parent) {
     });
   }
 
+  if (element.kind === 'translate') {
+      performTranslation(context, {x: element.x, y: element.y });
```

Comment:
Nitpick comments incoming! 😄 

This project uses 2 space indentation, also it looks like there's an extra space just before the closing brace.